Algorithms.cpp

```diff
@@ -99,11 +112,23 @@ int bubbleSort(int arr[],int length)
             }
         }
     }
+
+    cout << "Sorting with Bubble Sort...";
```

Comment:
Will look to unpack the "sorting" message to explain each step as done in the search algorithms